internal/codeintel/policies/matcher.go

```diff
@@ -256,6 +257,40 @@ func (m *Matcher) matchCommitsOnBranch(ctx context.Context, context matcherConte
 	return nil
 }
 
+// commitsUniqueToBranch returns a map from commits that exist on a particular
+// branch in the given repository to their committer date. This set of commits is
+// determined by listing `{branchName} ^HEAD`, which is interpreted as: all
+// commits on {branchName} not also on the tip of the default branch. If the
+// supplied branch name is the default branch, then this method instead returns
+// all commits reachable from HEAD.
+func commitsUniqueToBranch(ctx context.Context, gitserverClient gitserver.Client, repo api.RepoName, commitID api.CommitID, isDefaultBranch bool, maxCommitAge *time.Time) (map[api.CommitID]time.Time, error) {
+	rng := "HEAD"
```

Comment:
Can we change this to a pronounceable name? 

https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/doc/dev/background-information/languages/go.md#use-pronounceable-names

```suggestion
	range := "HEAD"
```